dali/pipeline/operators/util/optical_flow.cc

```diff
@@ -40,36 +40,75 @@ For values <=0, grid size is undefined. Currently only grid_size=4 is supported.
 
 DALI_REGISTER_OPERATOR(OpticalFlow, OpticalFlow<GPUBackend>, GPU);
 
+constexpr int kNOutputDims = 2;
+constexpr int kNInputDims = 4;
+
 
 template<>
 void OpticalFlow<GPUBackend>::RunImpl(Workspace<GPUBackend> *ws, const int) {
   if (enable_hints_) {
+    DALI_ENFORCE(false, "Hints currently ignored");
+    // TODO(mszolucha): hints currently ignored, add feature
     const auto &input = ws->Input<GPUBackend>(0);
     const auto &external_hints = ws->Input<GPUBackend>(1);
     auto &output = ws->Output<GPUBackend>(0);
+    size_t nsequences_ = input.ntensor();
+
+    of_lazy_init(frames_width_, frames_height_, depth_, ws->stream());
```

Comment:
Why lazy? This will slow down the first call of the operator, maybe it is better to move to the constructor.

Reply:
I think the issue here is that we don't know what stream to use when constructing the operator object. It's passed to us only at RunImpl time.

Reply:
We can create own stream for that op.

Reply:
The thing is also that we don't know frame sizes prior to RunImpl, so lazy is the only way actually